### ISSUE TITLE
Error summary links are wrong when used with the address component

### ISSUE THREAD

**Issue by Febakke** [Altinn/app-frontend-react#366]
### Description of the bug
[3.3.1 identifikasjon av feil (A)](https://www.uutilsynet.no/wcag-standarden/331-identifikasjon-av-feil-niva/116)
It seems like the error links created when using the address component only will link to the first field. (Gateadresse). So if you click on "du må fylle ut postnr" focus will be moved to "Gateadresse"
![Skjermbilde 2022-08-05 kl  12 57 11](https://user-images.githubusercontent.com/33222679/183064352-f5e6e7cc-259e-4154-b673-5ce2721488d3.png)

Bonus: Should we remove the error on the "poststed" field. There is no way for a user to edit this field, seems wrong to shift focus to a read-only field.

Update tested only error on postnr is not a part of summary 
![Skjermbilde 2023-01-11 kl  10 55 52](https://user-images.githubusercontent.com/33222679/211775448-6ea6f9fa-804a-4997-990d-e6769d4d4e33.png)

### Steps To Reproduce

1. Go to an app with required address component
2. Click on "send in"
3. Try the links in the error summary

**Comment by bjosttveit** [comment]
This could be a bit difficult to fix in the short term, because both validation messages and element focus is controlled at the _component_ level, and since the address component is a single component with multiple fields there is no mechanism for referencing a sub-field inside of the component. In order to fix this we would either need to hack something together or probably make some broader changes around these mechanisms.

**Comment by bjosttveit** [comment]
Should be possible to solve now, by extending the nodenavigation to accept a datamodel binding key to focus on as well component id. The binding key is now available in the validation object in the error report.
